string measurment
string sensor
bool value